debvader/position_optimization.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import scipy
+from scipy import optimize
+
+
+def position_optimization(
+    field_image,
+    output_image_mean_padded,
+    galaxy_distance_to_center,
+    method="scipy-minimize",
+):
+    """
+    Find shifts in the position of the deblended galaxy to minimize the mse between field_image
+
+    parameters:
+        field image: image of the entire field of galaxy to be deblended.
+        output_images_mean_padded: predicted image of the galaxy that is to be optimized.
+        galaxy_distances_to_center: distance of the predicted galaxy from the center, as detected by the detection algorithm.
+    """
+
+    assert method in ["scipy-minimize"]
+
+    if method == "scipy-minimize":
+
```

Comment:
I don't see the point in "preparing" for other methods.
Right now you only have a single method so don't go through the burden of adding checks and if statements. I would suggest you remove that part and add it again along with support for new optimization schemes.